policyengine_canada/variables/gov/provinces/nb/benefits/nbcb/nb_child_benefit_eligible_child.py

```diff
@@ -0,0 +1,16 @@
+from policyengine_canada.model_api import *
+
+
+class nb_child_benefit_eligible_child(Variable):
+    value_type = bool
+    entity = Person
+    label = "New Brunswick family benefit eligible child"
+    definition_period = YEAR
+    defined_for = ProvinceCode.NB
+
+    def formula(person, period, parameters):
+        age = person("age", period)
+        return (
```

Comment:
Also just checking this is < not <=
```suggestion
        p = parameters(period).gov.provinces.nb.benefits.nbcb
        return age < p.eligible_age
```

Reply:
Confirmed